src/replication.c

```diff
@@ -491,6 +492,46 @@ start_rebuild(void *buf, replica_t *replica, uint64_t data_len)
 	return ret;
 }
 
+/*
+ * Compare time..
+ * results: (T1 > T2) ? T1 : T2
+ */
+static int
```

Comment:
can this be made from int to bool, which makes this a conditional operator?

Reply:
yes